xml/System.IO/FileSystemWatcher.xml

```diff
@@ -105,7 +105,7 @@
 ## Examples  
  The following example creates a <xref:System.IO.FileSystemWatcher> to watch the directory specified at run time. The component is set to watch for changes in `LastWrite` and `LastAccess` time, the creation, deletion, or renaming of text files in the directory. If a file is changed, created, or deleted, the path to the file prints to the console. When a file is renamed, the old and new paths print to the console.  
   
- Use the <xref:System.Diagnostics> and <xref:System.IO> namespaces for this example.  
+ Use the <xref:System.IO> namespace for this example.  
```

Comment:
The sentence can be deleted entirely, since System.IO is imported in C#, VB, and C++ examples.